TypeScript error with `@types/react-native@0.70.4` due to usage of `Platform.isTVOS`

### Description

The following file uses `Platform.isTVOS` which was removed in the latest version of `@types/react-native`. This causes an error when running `tsc`.

https://github.com/software-mansion/react-native-screens/blob/4ed5ee333a2df4d5bf59162b311ec5253b3cbdee/src/native-stack/utils/getDefaultHeaderHeight.tsx#L25

```
node_modules/react-native-screens/src/native-stack/utils/getDefaultHeaderHeight.tsx:25:36 - error TS2551: Property 'isTVOS' does not exist on type 'PlatformIOSStatic'. Did you mean 'isTV'?
```

### Steps to reproduce

See description

### Snack or a link to a repository

Not needed

### Screens version

3.17.0

### React Native version

0.70.1

### Platforms

Android, iOS, Web, Windows, tvOS

### JavaScript runtime

_No response_

### Workflow

_No response_

### Architecture

_No response_

### Build type

_No response_

### Device

_No response_

### Device model

_No response_

### Acknowledgements

Yes

--- a/src/native-stack/utils/getDefaultHeaderHeight.tsx
+++ b/src/native-stack/utils/getDefaultHeaderHeight.tsx
@@ -22,7 +22,7 @@ export default function getDefaultHeaderHeight(
       statusBarHeight = 0;
     }
 
-    if (Platform.isPad || Platform.isTVOS) {
+    if (Platform.isPad || Platform.isTV) {
       headerHeight = isFromSheetModal ? formSheetModalHeight : 50;
     } else {
       if (isLandscape) {

